src/daft-core/src/ffi.rs

```diff
@@ -51,6 +51,22 @@ pub fn to_py_array(array: ArrayRef, py: Python, pyarrow: &PyModule) -> PyResult<
     Ok(array.to_object(py))
 }
 
+pub fn field_to_py(
+    field: &arrow2::datatypes::Field,
+    py: Python,
+    pyarrow: &PyModule,
+) -> PyResult<PyObject> {
+    let schema = Box::new(ffi::export_field_to_c(field));
+    let schema_ptr: *const ffi::ArrowSchema = &*schema;
```

Comment:
Why is it a schema ptr for a Field?

Can we add a comment here for future ref as well